### ISSUE TITLE
train  problem     卡在以下最后一行内容current training loss is :  0.01826123334467411 不动

### ISSUE THREAD

**Issue by gjy-code** [ShannonAI/mrc-for-flat-nested-ner#110]
train_zh_onto.sh: line 7: EXP-ID=22_1: command not found
Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.
Better speed can be achieved with apex installed from https://www.github.com/nvidia/apex.
Please notice that merge the args_dict and json_config ... ...
{
  "bert_frozen": "false",
  "hidden_size": 768,
  "hidden_dropout_prob": 0.2,
  "classifier_sign": "multi_nonlinear",
  "clip_grad": 1,
  "bert_config": {
    "attention_probs_dropout_prob": 0.1,
    "directionality": "bidi",
    "hidden_act": "gelu",
    "hidden_dropout_prob": 0.1,
    "hidden_size": 768,
    "initializer_range": 0.02,
    "intermediate_size": 3072,
    "max_position_embeddings": 512,
    "num_attention_heads": 12,
    "num_hidden_layers": 12,
    "pooler_fc_size": 768,
    "pooler_num_attention_heads": 12,
    "pooler_num_fc_layers": 3,
    "pooler_size_per_head": 128,
    "pooler_type": "first_token_transform",
    "type_vocab_size": 2,
    "vocab_size": 21128
  },
  "config_path": "/home/amax/work/gjy/mrcc/config/zh_bert.json",
  "data_dir": "/home/amax/work/gjy/mrcc/data_preprocess/example/zh_ontonotes4",
  "bert_model": "/home/amax/work/gjy/Bert-Ner-Demo-master/chinese_L-12_H-768_A-12",
  "task_name": null,
  "max_seq_length": 100,
  "train_batch_size": 16,
  "dev_batch_size": 16,
  "test_batch_size": 16,
  "checkpoint": 600,
  "learning_rate": 8e-06,
  "num_train_epochs": 10,
  "warmup_proportion": -1.0,
  "local_rank": -1,
  "gradient_accumulation_steps": 1,
  "seed": 2333,
  "export_model": true,
  "output_dir": "/home/amax/work/gjy/mrcc/export/zh_onto/mrc-ner-zh_onto-2020-05-12--100-8e-6-16-0.3",
  "data_sign": "zh_onto",
  "weight_start": 1.0,
  "weight_end": 1.0,
  "weight_span": 1.0,
  "entity_sign": "flat",
  "n_gpu": 1,
  "dropout": 0.3,
  "entity_threshold": 0.5,
  "data_cache": true
}
-*--*--*--*--*--*--*--*--*--*-
current data_sign: zh_onto
=*==*==*==*==*==*==*==*==*==*=
loading train data ... ...
62896
62896 train data loaded
=*==*==*==*==*==*==*==*==*==*=
loading dev data ... ...
17204
17204 dev data loaded
=*==*==*==*==*==*==*==*==*==*=
loading test data ... ...
17384
17384 test data loaded
数据已加载完毕！
load_model模型已加载完毕！
######################################################################
EPOCH:  0
-*--*--*--*--*--*--*--*--*--*--*--*--*--*--*-
current training loss is : 
0.01826123334467411
